packages/ui/src/kit/VLinkCollapse/VLinkCollapse.theme-default.scss

```diff
@@ -0,0 +1,18 @@
+.v-link-collapse.theme-default {
+  display: flex;
+  font-size: 14px;
+  text-decoration: none;
+  align-items: center;
+  justify-content: center;
+  color: var(--endpass-ui-color-primary-7);
+  cursor: pointer;
```

Comment:
no reason for this here

Reply:
try to remove `cursor: pointer` and you will see, that cursor will be default for `text selection` but not for default as `link`

Reply:
this is very strange, because a tag always have pointer cursor by default
can't see any reason to add this line here